File: src/main/java/dev/infernity/polarisplugin/polarisplugin/tasks/EverySecondTask.java
package dev.infernity.polarisplugin.polarisplugin.tasks;

import org.bukkit.Bukkit;
import org.bukkit.entity.Player;
import org.bukkit.scheduler.BukkitRunnable;

public class EverySecondTask extends BukkitRunnable {
    @Override
    public void run(){
        for (Player player : Bukkit.getOnlinePlayers()){
            if (player.getSaturation() > 20){
                player.setSaturation(20f);
            }
            if (player.getFoodLevel() > 20){
                player.setFoodLevel(20);
            }
        };
    }
}

File: src/main/java/dev/infernity/polarisplugin/polarisplugin/tasks/EverySecondTask.kt
package dev.infernity.polarisplugin.polarisplugin.tasks

import org.bukkit.Bukkit
import org.bukkit.scheduler.BukkitRunnable

class EverySecondTask : BukkitRunnable() {
    override fun run() {
        for (player in Bukkit.getOnlinePlayers()) {
            if (player.saturation > 20) {
                player.saturation = 20f
            }
            if (player.foodLevel > 20) {
                player.foodLevel = 20
            }
        }
    }
}